string age
string gender
string skin_color
string hair_style
string glasses
string clothes_color
string clothes
string pose
